wallets/core/src/wallet.ts

```diff
@@ -12,9 +12,14 @@ export type EventHandler = (
   event: Events,
   value: any,
   coreState: State,
-  supportedChains: BlockchainMeta[]
+  meta: EventMeta
```

Comment:
To be more consistent with https://github.com/rango-exchange/rango-client/pull/427/files#r1382562861 , If we rename it the meta to `info` is better.

`info: EventInfo`

Reply:
My suggestion is to change `EventInfo` to something `WalletInfo` or something like that. because this model is also used in other sections (e.g. wallets.ts private field) which are not related to Events.